src/simtools/visualization/simtel_event_plots.py

```diff
@@ -0,0 +1,612 @@
+#!/usr/bin/python3
+"""Plots for light emission (flasher/calibration) sim_telarray events."""
+
+from __future__ import annotations
```

Comment:
My suggestion is not to use any future features but stick to our current python version.